packages/aws-cdk-lib/aws-ecs/lib/base/task-definition.ts

```diff
@@ -194,8 +194,10 @@ export interface TaskDefinitionProps extends CommonTaskDefinitionProps {
    * The process namespace to use for the containers in the task.
    *
    * Only supported for tasks that are hosted on AWS Fargate if the tasks
-   * are using platform version 1.4.0 or later (Linux).
-   * Not supported in Windows containers.
+   * are using platform version 1.4.0 or later (Linux). Only the TASK option
+   * is supported for Linux-based Fargate containers. Not supported in Windows
+   * containers. If pidMode is specified for a Fargate task, then
+   * runtimePlatform.operatingSystemFamily must also be specified.
    *
```

Comment:
nit: we can put the doc link here for reference.

Reply:
Sure!  I added a link to it.